src/style.css

```diff
@@ -0,0 +1,273 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  font-family: 'Nunito Sans', sans-serif;
+  background-color: hsl(207, 26%, 17%);
+  color: hsl(0, 0%, 100%);
+}
+
+header {
+  background-color: hsl(209, 23%, 22%);
+  display: flex;
+  padding: 1% 10%;
+  justify-content: space-between;
+}
+
+header h2 {
+  font-size: 30px;
+  letter-spacing: 2px;
+}
+
+#logo {
+  width: 90px;
+  height: 90px;
+}
+
+.like {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  transition: all 0.5s ease-in-out;
+}
+
+.like:active {
+  width: 50px;
+  height: 50px;
+}
+
+.likes {
+  font-size: 14px;
+}
+
+.img-holder {
+  height: 180px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+}
+
+ul {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin: 0 auto;
+  gap: 4rem 3.5rem;
+}
+
+#countries {
+  display: flex;
+  align-self: center;
+  gap: 1rem;
+  align-items: center;
+}
+
+article h2 {
+  margin-bottom: 0.5rem;
+  font-weight: 600;
+}
+
+#countries h2 {
+  padding-bottom: 2px;
+  border-bottom: 3px solid gray;
+  font-size: 25px;
+  margin-top: 2rem;
+}
+
+.info {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 5%;
+}
+
+li {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 0.8rem;
+  background-color: hsl(209, 23%, 22%);
+  border-radius: 8px;
+  width: 300px;
+}
+
+.comment-btn {
+  padding: 2% 2%;
+  margin-bottom: 5%;
+  width: 80%;
+  font-size: 1rem;
+  letter-spacing: 2px;
+  box-shadow: 1px 1.5px 3px #000;
+  background-color: hsl(207, 36%, 37%);
+  align-self: center;
+  border-radius: 5%;
+  color: #fff;
+  position: relative;
+  bottom: 0;
+}
+
+article h3 {
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 23px;
+}
+
+article span {
+  font-weight: 300;
+  font-size: 14px;
+}
+
+footer {
+  width: 100%;
+  align-self: center;
+  border-radius: 0.5%;
+  border: 1px solid black;
+  background-color: hsl(209, 23%, 22%);
+  text-align: center;
+  padding: 1rem 0;
+}
+
+.button1,
+#search {
+  width: 30%;
+  padding: 0.7% 2.5%;
+  border-radius: 8px;
+  border: 1px solid hsl(207, 26%, 17%);
+  background-size: 30px 25px;
+  background-repeat: no-repeat;
+  background-position: 3% center;
+  padding-left: 50px;
+  background-color: #fff;
+}
+
+#search {
+  background-color: #fff;
+}
+
+/* Pop up Code  */
+#popup-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#name-input {
+  width: 20em;
+  height: 2rem;
+}
+
+.popup-window {
+  top: 15%;
+  left: 20%;
+  right: 20%;
+  height: 80%;
+  display: none;
+  grid-template-rows: 1fr 1fr 1fr;
+  background-color: hsl(209, 23%, 22%);
+  position: fixed;
+  backdrop-filter: blur(10px);
+}
```

Comment:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 
- [OPTIONAL] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to make a good contrast 🎨 👨🏻‍🎨 
- Note: Adding these features will make your design look great 😉 👍🏻

Reply:
Thank you for suggesting that, we have implemented it 😄